codegra_fs/cgfs.py

```diff
@@ -2017,6 +2017,117 @@ def check_version() -> None:
 
 def main() -> None:
     global cgapi
+    logger.info('Mounting... ')
```

Comment:
This shouldn't be added in this pull request.

Reply:
Fixed!